browser/net/brave_request_handler.cc

```diff
@@ -107,6 +97,19 @@ void BraveRequestHandler::SetupCallbacks() {
       base::BindRepeating(brave::OnBeforeStartTransaction_BraveServiceKey);
   before_start_transaction_callbacks_.push_back(start_transaction_callback);
 
+#if BUILDFLAG(ENABLE_IPFS)
+  if (base::FeatureList::IsEnabled(ipfs::features::kIpfsFeature)) {
```

Comment:
the order these things run in is actually important in some cases so can we keep it in one block, but move it back to the original location? I want to avoid unintentionally creating any problems here